### PR TITLE
Feature/c100-rebuild-PRLC100 #194

### PR DESCRIPTION
**Pull request by giorgi-hmcts**
### Change description ###
- C100 Rebuild - **Does your situation qualify for an urgent hearing?** page
- getNextStep() - for now, both 'YES' and 'NO' answers redirects the user the same current page: 
   '/c100-rebuild/typeofhearing/urgent'

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/typeofhearing/urgent/content.ts [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    urgentHearingReasons: {
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/typeofhearing/urgent/content.ts:59]: please update the field name as typeOfHearing_<fieldName>

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/urls.ts [hunk 2]
```diff
@@ -246,3 +246,6 @@ export const C100_MIAM_MEDIATOR_CONFIRMAION: PageLink = `${C100_MIAM}/mediator-c
 export const C100_MIAM_URGENCY: PageLink = `${C100_MIAM}/urgency`;
 export const C100_MIAM_INFO: PageLink = `${C100_MIAM}/miam-info`;
 export const C100_MIAM_VALID_REASON: PageLink = `${C100_MIAM}/valid-reason`;
+
+/** @C100 Urgency */
+export const C100_URGENT_HEARING: PageLink = `${C100_URL}/typeofhearing/urgent`;
```
Comment (yogendraU) [inline src/main/steps/urls.ts:251]: Please change the folder name from typeofhearing to hearing-urgency

File: src/main/steps/c100-rebuild/typeofhearing/urgent/template.njk [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-top-7">{{title}}</h1>
+<p class="govuk-body">{{paragraph1}}</p>
+<p class="govuk-body">{{paragraph2}}</p>
+<p class="govuk-body">{{paragraph3}}</p>
+
+{{ govukWarningText({
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/typeofhearing/urgent/template.njk:10]: Please move this text message to content.ts

You can refer to warningMessage in src/main/steps/c100-rebuild/safety-concerns/concerns-for-safety/content.ts

File: src/main/steps/c100-rebuild/typeofhearing/urgent/template.njk [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-top-7">{{title}}</h1>
+<p class="govuk-body">{{paragraph1}}</p>
+<p class="govuk-body">{{paragraph2}}</p>
+<p class="govuk-body">{{paragraph3}}</p>
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/typeofhearing/urgent/template.njk:8]: Please check if we can move paragraph1, paragraph2, paragraph3 inside content.ts fields

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/typeofhearing/urgent/content.ts (hunk 1 above)
Comment (channa-patil) [reply src/main/steps/c100-rebuild/typeofhearing/urgent/content.ts]: @yogendraU We can use prefix 'toh_' for type of hearing fields.
@giorgi-hmcts Please use prefix 'toh_'

Comment (channa-patil) [reply src/main/steps/c100-rebuild/typeofhearing/urgent/content.ts]: As discussed & agreed. We will use prefix 'hu_'

**Review by channa-patil — APPROVED** [review]

File: src/main/steps/c100-rebuild/hearing-urgency/urgent/template.njk [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+<h1 class="govuk-heading-xl govuk-!-margin-top-7">{{title}}</h1>
+{% for paragraph in paragraphs %}
+        <p class="govuk-body">{{paragraph | safe}}</p>
+{% endfor %}
+
+{{ govukWarningText(getContent(warningText)) }}
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/hearing-urgency/urgent/template.njk:10]: This could have been moved to content.ts

**Pull request merged**